WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsView.swift

```diff
@@ -14,73 +14,79 @@ struct PointOfSaleSettingsView: View {
+                    .safeAreaInset(edge: .bottom) {
+                        Button {
+                            selection = .help
+                        } label: {
+                            HStack {
+                                Image(systemName: SidebarNavigation.help.icon)
                                     .font(.posBodyLargeRegular())
                                     .foregroundStyle(selection == .help ? .white : .primary)
-                                Text(SidebarNavigation.help.subtitle)
-                                    .font(.posBodyMediumRegular())
-                                    .foregroundStyle(selection == .help ? .secondary : .secondary)
+                                VStack(alignment: .leading) {
+                                    Text(SidebarNavigation.help.title)
+                                        .font(.posBodyLargeRegular())
+                                        .foregroundStyle(selection == .help ? .white : .primary)
+                                    Text(SidebarNavigation.help.subtitle)
+                                        .font(.posBodyMediumRegular())
+                                        .foregroundStyle(selection == .help ? .secondary : .secondary)
+                                }
+                                Spacer()
                             }
-                            Spacer()
+                            .padding(.vertical, POSPadding.small)
+                            .padding(.horizontal, POSPadding.medium)
+                            .contentShape(Rectangle())
                         }
-                        .padding(.vertical, POSPadding.small)
-                        .padding(.horizontal, POSPadding.medium)
-                        .contentShape(Rectangle())
+                        .buttonStyle(.plain)
+                        .background(
+                            RoundedRectangle(cornerRadius: POSCornerRadiusStyle.large.value, style: .continuous)
+                                .fill(selection == .help ? Color.accentColor : Color.clear)
+                        )
                     }
-                    .buttonStyle(.plain)
-                    .background(
-                        RoundedRectangle(cornerRadius: POSCornerRadiusStyle.large.value, style: .continuous)
-                            .fill(selection == .help ? Color.accentColor : Color.clear)
-                    )
                 }
-            }
-            Group {
-                switch selection {
-                case .store:
-                    PointOfSaleSettingsStoreDetailView()
-                case .hardware:
-                    PointOfSaleSettingsHardwareDetailView()
-                case .help:
-                    PointOfSaleSettingsHelpDetailView()
-                default:
-                    EmptyView()
+                .frame(width: geometry.size.width * Constants.sidebarWidthFraction)
+                Group {
```

Comment:
Wwe could avoid using `Group` if you would move the switch into a separate `@ViewBuilder`:

```swift
    @ViewBuilder
    private var detailView: some View {
        switch selection {
        case .store:
            PointOfSaleSettingsStoreDetailView()
        case .hardware:
            PointOfSaleSettingsHardwareDetailView()
        case .help:
            PointOfSaleSettingsHelpDetailView()
        default:
            EmptyView()
        }
    }
```